### ISSUE TITLE
Problem with Imported Songs and Controller Settings

### ISSUE THREAD

**Issue by RealLocutus** [piotrulos/MSCModLoader#139]
Please read FAQ before!
#### Info
* **OS version/build:** Windows 7
* **Architecture:** 64bit
* **MSCLoader version:** v1.2.5
* **Type of issue:** bug
* 
#### Description
I got two problems which I think are caused by MSC Loader. They work in Vanilla Game and as well with the Mod Loader Pro.

1st:
I use a lot of custom songs which I import through the built in function to the radio.
The Problem is that the songs don't stay imported when you Save or Quit to the menu. 
You have to import them every time you go to the menu. 
In Vanilla they stay imported until you Quit the game completely.

2nd:
I use a Xbox One Controller wired via USB. The settings for the Deadzones are not saved correctly.
I have to set the Deadzone everytime I load into the game.

I recently changed from Mod Loader Pro to MSC Loader again - bc MOP Mod required it. I did not change any mods.
So above mentioned issues are probably caused by MSCLoader itself. Please fix, if possible. Thanks.

#### Attachments
none

**Comment by piotrulos** [comment]
1. Someone still uses that broken vanilla import songs? There is plenty of mods that fixes that you are not required to import anything.

2. ModLoader doesn't change anything related to controllers or saving stuff.

3. No output_log.txt = not a valid bug repport.

**Comment by RealLocutus** [comment]
1. Yeah I use it? Why not? Works like it should? I use it bc I import it through the Radio folder and not as a CD. I want the music in every car that has a Radio and not only in the Satsuma when I got the CD Radio. And I know mods like the "CD Player enhanced" but they do not import it like a radio.

2. OK then tell me why the settings for the controller are saved correctly when I use the vanilla game without any mods and even when I use the other ModloaderPro? I recently changed back to MSC Loader bc MOP Mod required it. So apart from the newer version of MOP Mod I use the same mods as before. So the only difference is the Mod Loader. So why is the game now not saving the Deadzone settings for the controller anymore?

3. here you have the file
[output_log.txt](https://github.com/piotrulos/MSCModLoader/files/8029331/output_log.txt)
